frost-core/src/lib.rs

```diff
@@ -191,6 +191,32 @@ pub trait Ciphersuite: Copy + Clone {
     ///
     /// [H4]: https://github.com/cfrg/draft-irtf-cfrg-frost/blob/master/draft-irtf-cfrg-frost.md#cryptographic-hash
     fn H4(m: &[u8]) -> <<Self::Group as Group>::Field as Field>::Scalar;
+
+    /// Verify a signature for this ciphersuite. The default implementation uses the "cofactored"
+    /// equation (it multiplies by the cofactor returned by [`Group::cofactor()`]).
+    /// You may override this to provide a tailored implementation, but it must also multiply
+    /// by the cofactor to comply with the RFC.
+    fn VerifySignature(
```

Comment:
We can match the standard Rust naming for this function, like `generate()`, etc

```suggestion
    fn verify_signature(
```

Reply:
done in 27a5b93